vid/src/utils/bytes_to_field.rs

```diff
@@ -105,11 +105,11 @@ where
     fn next(&mut self) -> Option<Self::Item> {
         let mut elem_bytes = Vec::with_capacity(self.elem_byte_capacity);
         for _ in 0..elem_bytes.capacity() {
-            if let Some(byte) = self.bytes_iter.next() {
+            match self.bytes_iter.next() { Some(byte) => {
                 elem_bytes.push(*byte.borrow());
-            } else {
+            } _ => {
```

Comment:
using `None` would be better here than `_`?

Reply:
It seems we can still compile even if we revert that automatic fix. I don't think it should change the behavior if we leave it as it was, so I reverted the change.